index.js

```diff
@@ -72,6 +74,27 @@ module.exports = {
       console.log()
 
       await deleteSentryConfig()
+
+      if (!shouldDeleteSourceMaps) {
+        console.log('Source map files retained.')
+      } else {
+        console.log('Removing source map files.')
+        await new Promise(resolve => {
+          const glob = require('glob');
+          glob(`${sourceMapPath}/**/*.map`, (er, files) => {
+            if (!er && files) {
+              Promise
+                .all(files
+                  .map(file => deleteFile(file)))
+                .catch(e => console.log('error deleting source map files', e))
+                .finally(() => resolve(true));
+            } else {
+              console.log('could not find source map files', er);
+              resolve(false);
+            }
+          })
```

Comment:
Since the `resolve` values aren't being used for anything, it might be a bit simpler with something like this:
```js
if (inputs.deleteSourceMaps) {
  console.log("Removing source map files.");
  const files = glob.sync(`${sourceMapPath}/**/*.map`);
  const deleteFileErrorHandler = (e) => console.log("Error deleting source map file:", e);
  const deleteFilePromises = files.map((file) =>
    deleteFile(file).catch(deleteFileErrorHandler)
  );
  await Promise.all(deleteFilePromises);
}
```

Reply:
This also would remove the "Source map files retained." log, which I feel is a bit redundant.

Reply:
agree, your version is simpler.  old glob package didn't have the sync option, hence the promise/resolve/etc.